backend/src/resolvers/user.resolver.ts

```diff
@@ -8,3 +10,25 @@ export const UserQueryResolver: Pick<QueryResolvers.Type
         return dataSources.databaseAPI.getAllUsers();
     }
 };
+
+export const UserMutationResolver: Pick<MutationResolvers.Type
+    , 'login'
+    | 'signup'
+    > = {
+    // @ts-ignore
+    login: (parent, {email, password}, {dataSources}) => {
+        const user = dataSources.databaseAPI.getUserPerEmail(email);
+        if (!user) return null;
+        if (user.password !== generateSHA512Hash(password)) return null;
+        return createToken(user.id);
```

Comment:
⭐ For an implemented login feature using JWT. (in function createToken JWT is used)